tower-grpc-interop/src/client.rs

```diff
@@ -641,14 +640,13 @@ impl TestClients {
     fn unimplemented_method_test(&mut self)
             -> impl Future<Item=Vec<TestAssertion>, Error=Box<Error>> {
```

Comment:
This could be done later, but I think it is possible that this PR removes the need for `Box<Error>`; the reason that is used is because there were multiple error types because of the generic `Error`.